ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml

```diff
@@ -1505,6 +1505,14 @@ tests:
     test:
     - chain: openshift-e2e-test-hypershift-qe
     workflow: cucushift-installer-rehearse-azure-aks-hypershift-etcd-disk-encryption-guest
+- as: azure-aks-hypershift-ext-oidc-guest-f7
+  cron: 23 19 5,12,19,27 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;~NonPreRelease&;~HyperShiftMGMT&;~MicroShiftOnly&;~NonHyperShiftHOST&;~Serial&;~Disruptive&
+      TEST_TIMEOUT: "30"
+    workflow: cucushift-installer-rehearse-azure-aks-hypershift-ext-oidc-guest
```

Comment:
I would suggest to add/override `test` there, though the test is already defined in the workflow. Teammates generally comes to this folder to check config things.

Reply:
You can add a new test chain if there are multiple test steps included and you don't want to copy/paste them for several places/versions.

Reply:
Updated.